### PR TITLE
add a note for version compatibility

### PR DESCRIPTION
**Pull request by rajatchopra**
Fix for https://bugzilla.redhat.com/show_bug.cgi?id=1459388

### CODE REVIEW

File: install_config/router/f5_router.adoc [hunk 1]
```diff
@@ -210,6 +210,16 @@ xref:../../architecture/additional_concepts/f5_big_ip.adoc#architecture-f5-nativ
 Native Integration] section of the Routes topic.
 ====
 
+[NOTE]
+====
+Only *F5 BIG-IP®* appliance version 12.x and above will work with the Native integration
+presented in this section. Also, one needs a 'sdn-services' add-on license for the
```
Comment (ahardin-rh) [inline install_config/router/f5_router.adoc]: s/Also, one needs/You also need
s/'sdn-services'/`sdn-services`

File: install_config/router/f5_router.adoc [hunk 2]
```diff
@@ -210,6 +210,16 @@ xref:../../architecture/additional_concepts/f5_big_ip.adoc#architecture-f5-nativ
...
+integration to work properly.
+For version 11.x, please follow the instructions to set up the
```
Comment (ahardin-rh) [inline install_config/router/f5_router.adoc]: s/please follow/follow

**Comment by rajatchopra** [comment]
@ahardin-rh Thanks for the review. Fixed and updated.
Backport PR: https://github.com/openshift/openshift-docs/pull/4581

**Comment by ahardin-rh** [comment]
Thank you!

**Comment by ahardin-rh** [comment]
Removing milestones since https://github.com/openshift/openshift-docs/pull/4582 add this to the appropriate branch

**Pull request merged**